junit5/README.md

```diff
@@ -277,3 +282,42 @@ class AddBeanTest {
     }
 }
 ```
+
+## Additional Configuration
+
+This section describes any additional configuration options this extension offers.
+
+### Explicit Parameter Injection
+
+As mentioned above, Weld is greedy when it comes to parameter injection.
+It will claim the ability to resolve any parameter which is known as a bean type inside running CDI container.
+This is mainly for usability, as it would be annoying to constantly type additional annotations to mark which parameter should be injected and which should be left alone.
+
+However, we are aware that this might cause trouble if more extensions are competing for parameter resolution.
+In such case, you can turn on explicit parameter resolution and Weld will only resolve parameters which have at least one `javax.inject.Qualifier` annotation on them.
+There are two ways to enable it; firstly, you can do it globally, through system property - `org.jboss.weld.junit5.explicitParamInjection=true`
+This property is also available as a constant in our extension class, e.g. you can use `org.jboss.weld.junit5.WeldJunit5Extension.GLOBAL_EXPLICIT_PARAM_INJECTION`.
+Secondly, you can use `@ExplicitParamInjection` on your method, or test class.
+In case of test class this annotation will enforce the presence on qualifiers on all methods.
+
+Let's have a look at it:
+
+```java
+@EnableWeld
+@ExplicitParamInjection // all methods will now require explicit parameters
+class ExplicitParamInjectionTest {
+
+    @Test
+    public void testThatParamsAreNotResolvedByWeld(Foo foo) {
+        // Weld will not attempt to resolve Foo, hence this test will fail unless there is another extension resolving it
+    }
+
+    @Test
+    public void testThatParamsAreResolvedByWeld(@Default Foo foo, @MyQualifier Bar bar) {
+        // Weld will resolve both of the parameters
+    }
+}
+```
+
+As most of you know, if you want to inject a bean where you would normally not use any qualifier, you can do that using `@Default` qualifier (as shown in the code above).
```

Comment:
Hehe, "most of you" ;-)

Reply:
I was just politely suggesting they might want to give it a read :)